packages/screens/LaunchpadERC20/component/LaunchpadERC20Banner.tsx

```diff
@@ -0,0 +1,43 @@
+import React from "react";
+import { Image, View } from "react-native";
+
+import LaunchpadERC20BannerImage from "../../../../assets/banners/launchpadERC20.jpg";
+import LogoSimpleSvg from "../../../../assets/icons/logo-simple.svg";
+
+import { BrandText } from "@/components/BrandText";
+import { SVG } from "@/components/SVG";
+import { SpacerColumn } from "@/components/spacer";
+import { useMaxResolution } from "@/hooks/useMaxResolution";
+import { fontSemibold16, fontSemibold28 } from "@/utils/style/fonts";
+
+export const LaunchpadERC20Banner: React.FC = () => {
+  const { width } = useMaxResolution();
+  const isSmallScreen = width < 500;
```

Comment:
Just a suggestion: I think we should centralize the screen "breakpoints" if possible :D
I mean, we can write a hook to return if the given screen is "small, medium, large"

Reply:
[5b39cf7](https://github.com/TERITORI/teritori-dapp/pull/1276/commits/5b39cf7ca8f7a7767589fae0ce447ea440ae2551)

Reply:
@hthieu1110 don't worry, since :
Here https://deploy-preview-1276--teritori-dapp.netlify.app/launchpad-erc20, @MikaelVallenet just reused the code from https://deploy-preview-1276--teritori-dapp.netlify.app/launchpad/apply

The component `LaunchpadERC20Banner` is the same as `LaunchpadBanner` https://github.com/TERITORI/teritori-dapp/blob/68f9e795d80541347a215c49e350ee678079486a/packages/screens/Launchpad/components/LaunchpadBanner.tsx#L13

BTW: For this case, we have 2 solutions :
- Add props to `LaunchpadBanner` to make it reusable in Launchpad ERC20 UI
- Make a new banner component that is almost the same than `LaunchpadBanner`. Code duplication, but it's more clear.

The 2d solution (current) is a good choice if we really want to separate Launchpad and LaunchpadERC20  codes. I think we should, to increase code lisibility.

BTW: We have some breakpoints here: `packages/utils/style/layout.ts`